src/AzureDBExperiences.ts

```diff
@@ -101,7 +101,7 @@ export function getExperienceQuickPick(api: API): IAzureQuickPickItem<Experience
 
 export function getExperienceQuickPickForAttached(api: API): IAzureQuickPickItem<Experience> {
     const exp = getExperienceFromApi(api);
-    return { label: exp.shortName, description: exp.description, data: exp };
+    return { label: exp.longName, description: exp.description, data: exp };
```

Comment:
Why this change and where is it effective?

Reply:
It's accessed when a user attempts to attach an account using the "Attached Database Accounts" path. It's used to display the options available. Previously, the `shortName` was shown.

<img width="581" alt="image" src="https://github.com/user-attachments/assets/ef566e09-67d2-4539-8d8b-339a7a8abb07">